src/unsafe-rust/unsafe-traits.md

```diff
@@ -8,36 +8,35 @@ Like with functions, you can mark a trait as `unsafe` if the implementation must
 guarantee particular conditions to avoid undefined behaviour.
 
 For example, the `zerocopy` crate has an unsafe trait that looks
-[something like this](https://docs.rs/zerocopy/latest/zerocopy/trait.AsBytes.html):
+[something like this](https://docs.rs/zerocopy/latest/zerocopy/trait.IntoBytes.html):
 
 ```rust,editable
-use std::mem::size_of_val;
+use std::mem;
 use std::slice;
```

Comment:
Nice change and now we can import both with one line (vertical space is a precious resource on our slides):

```suggestion
use std::{mem, slice};
```